src/pages/docs/tutorials/setup/index.md

```diff
@@ -72,26 +72,28 @@ App
 
 ### Configuring the Application
 
-Your application is configured using the `package.json` file. We'll only cover the fields you most likely want to change here and leave a more [complete discussion of the `package.json`](#package.json-In-Depth) for further reading.
+Your application is configured using the `package.json` file. We'll only cover the fields you most likely want to change here and leave a more [complete discussion of the `package.json`](https://docs.npmjs.com/cli/v6/configuring-npm/package-json) for further reading.
 
 * `"name"`, `"version"`, `"description"`, `"author"`, `"license"` - Application meta-data. These fields do not affect the build of your application but should be updated to reflect your application's details.
-* `"enact"` - The Enact-specific configuration block. By default, it contains intelligent defaults for an Enact Moonstone application. You *should* add a `"title"` field within this block to specify your application's title which will be included in the generated `index.html` during the build.
+* `"enact"` - The Enact-specific configuration block. By default, it contains intelligent defaults for an Enact Sandstone application. You *should* add a `"title"` field within this block to specify your application's title which will be included in the generated `index.html` during the build.
+
+Current `enact create` command creates a Moonstone based application as defaults. To use the application as Sandstone base, we'll change `description`, `theme`, and `dependencies` in the `package.json` file. In the future, it will create Sandstone based application as defaults. The changed `package.json` is shown below.
```

Comment:
I think if you plan to remove this when it's merged to develop, it's okay not to add this explanation.

Reply:
Yep, I'll remove that.